tests/utils/test_db.py

```diff
@@ -35,9 +34,6 @@ def test_database_schema_and_sqlalchemy_model_are_in_sync(self):
         all_meta_data = MetaData()
         for (table_name, table) in airflow_base.metadata.tables.items():
             all_meta_data._add_table(table_name, table.schema, table)
-        if RBAC:
```

Comment:
nit: can we also remove the comment `# combine Airflow and Flask-AppBuilder (if rbac enabled) models`